Creating a file for sandiego-ca

kshafer1 opened the pull request:
This pull request resolves #34

kshafer1 committed:
Created a skeleton for the sandiego-ca.md file

this is my extended commit message - blah blah blah

rsujatinov commented on places/sandiego-ca.md:
```diff
@@ -0,0 +1,7 @@
+## San Diego, CA
```
please add more information

kshafer1 committed:
Added information to at least one category

Added some information - this is the extended
information from the commit
blah blah blah
